source/PlayState.hx

```diff
@@ -1,12 +1,14 @@
 package;
 
 import flixel.FlxState;
+import haxe.display.Protocol.HaxeResponseErrorData;
```

Comment:
This import isn't necessary. To keep things clean, it'd be better to remove it. Having it around doesn't break functionality, though, so I won't block my approval on it.